frontend/src/components/ui/classes.ts

```diff
@@ -6,7 +6,7 @@ export const button = {
     'bg-transparent border border-2 rounded-sm  px-6 h-[34px] border-white text-white text-center block  text-xs text-decoration-none whitespace-nowrap hover:opacity-80 disabled:opacity-30',
 
   primaryButton:
-    'bg-transparent border border-2 rounded-sm  px-6 h-[34px] border-primaryBlue text-center block  text-xs text-decoration-none whitespace-nowrap hover:opacity-50 text-primaryBlue hover:text-primaryBlue disabled:opacity-30',
+    'bg-transparent border border-2 rounded-sm  px-6 h-[34px] border-primaryBlue text-center block font-semibold text-xs text-decoration-none whitespace-nowrap hover:opacity-50 text-primaryBlue hover:text-primaryBlue disabled:opacity-30',
```

Comment:
Does the BC Gov Sans have a semi bold option?

Reply:
It seems to work ok

Reply:
nice